feature/login/src/main/java/com/epmedu/animeal/login/profile/presentation/FinishProfileScreenUi.kt

```diff
@@ -0,0 +1,187 @@
+package com.epmedu.animeal.login.profile.presentation
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusDirection
+import androidx.compose.ui.focus.FocusManager
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.extensions.DAY_MONTH_YEAR_DOT_FORMATTER
+import com.epmedu.animeal.extensions.formatDateToString
+import com.epmedu.animeal.foundation.button.AnimealShortButton
+import com.epmedu.animeal.foundation.dialog.DatePickerDialog
+import com.epmedu.animeal.foundation.input.BirthDateInput
+import com.epmedu.animeal.foundation.input.PhoneNumberInput
+import com.epmedu.animeal.foundation.input.TextInputField
+import com.epmedu.animeal.foundation.spacer.HeightSpacer
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.foundation.topbar.BackButton
+import com.epmedu.animeal.foundation.topbar.TopBar
+import com.epmedu.animeal.login.profile.presentation.ui.ProfileEvent
+import com.epmedu.animeal.resources.R
+
+@Suppress("LongMethod")
```

Comment:
You can extract some composables in separate methods instead of suppressing detekt issue

Reply:
Done, dis separate composables